core/src/main/java/com/datastrato/gravitino/authorization/AuthorizationHook.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.authorization;
```

Comment:
I would suggest to move this package to `connector`, all the developer facing interface is under connector package, what do you think?

Reply:
DONE.